python/README.md

```diff
@@ -15,6 +15,26 @@
  - limitations under the License.
  -->
 
-# Iceberg
-A python implementation of the Iceberg table format.
-See the project level README for more details: https://github.com/apache/incubator-iceberg
+# Iceberg Python
+
+Iceberg is a python library for programatic access to iceberg table metadata as well as data access. The intention is to provide a functional subset of the java library.
+
+## Getting Started
+
+We are not currently publishing to PyPi so the best way to install the library is to clone the git repo and do a pip install -e
+
+```
+git clone https://github.com/apache/incubator-iceberg.git
+cd incubator-iceberg/python
+pip install -e .
```

Comment:
This doesn't quite resolve #323 because it doesn't document how to run python tests.

Reply:
added tox instructions, let me know if that looks ok

Reply:
Did the other changes to this file make it? Looks like the empty line is still there and I don't see the test instructions.